__tests__/app.test.js

```diff
@@ -122,12 +122,66 @@ describe('/api/articles', () => {
             expect(dates).toEqual(sortedDates);
         })
     });
-    test('GET: 404 responds with corrent error when given a route that does not exist', () => {
+    test('GET: 404 responds with correct error when given a route that does not exist', () => {
         return request(app)
         .get('/api/notaroute')
         .expect(404)
         .then((response) => {
             expect(response.statusCode).toBe(404);
         })
     });
+});
+
+describe('/api/articles/:article_id/comments', () => {
+    test('GET: 200 responds with an array of comments for given article', () => {
+        const id = 3;
+        return request(app)
+        .get(`/api/articles/${id}/comments`)
+        .expect(200)
+        .then((response) => {
+            const comments = response.body
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
```

Comment:
Since we're making a request to the article id of 3, you can expect that the returned comments will have an article id of 3